userspace/falco/configuration.cpp

```diff
@@ -252,6 +252,10 @@ void falco_configuration::init(string conf_filename, list<string> &cmdline_optio
 	{
 		throw logic_error("Error reading config file(" + m_config_file + "): the maximum consecutive timeouts without an event must be an unsigned integer > 0");
 	}
+
+	m_metadata_download_max_mb = m_config->get_scalar<uint32_t>("metadata_download", "max_mb", 100);
```

Comment:
Can we please cap these values (both lower and upper bounds)?

Reply:
It has been fixed here 
https://github.com/falcosecurity/falco/pull/1667/files#diff-9bd683a06f316f8dab7df7c882f9d1857df11de6ae476356e5f2f86068891157R257